dhcpv4/client.go

```diff
@@ -42,12 +46,20 @@ func NewClient() *Client {
 	}
 }
 
-// MakeRawBroadcastPacket converts payload (a serialized DHCPv4 packet) into a
-// raw packet suitable for UDP broadcast.
+// MakeRawBroadcastPacket leverages MakeRawPacket to create a raw packet suitable
+// for UDP broadcast.
 func MakeRawBroadcastPacket(payload []byte) ([]byte, error) {
```

Comment:
Is this being used? If not I would remove it.

Reply:
before removing it, this should be deprecated - grep "deprecated" in the tree to see other similar cases

Reply:
Will do.

Reply:
I think this was mostly being used internally but a "deprecated" message sounds ok.